h2o-docs/src/product/data-science/upliftdrf.rst

```diff
@@ -0,0 +1,470 @@
+
+  ``nbins`` and ``nbins_top_level`` are both for numerics (real and integer). ``nbins_top_level`` is the number of bins Uplift DRF uses at the top of each tree. It then divides by 2 at each ensuing level to find a new number. ``nbins`` controls when Uplift DRF stops dividing by 2.
+
+-  **How is variable importance calculated for Uplift DRF?**
+
+  Variable importance is not supported for Uplift DRF.
+
+-  **How is column sampling implemented for Uplift DRF?**
+
+  For an example model using:
+
+  -  100 columns
+  -  ``col_sample_rate_per_tree`` is 0.602
+  -  ``mtries`` is -1 or 7 (refers to the number of active predictor columns for the dataset)
+
+  For each tree, the floor is used to determine the number of columns that are randomly picked (for this example, (0.602*100)=60 out of the 100 columns). 
+
+  For classification cases where ``mtries=-1``, the square root is randomly chosen for each split decision (out of the total 60 - for this example, (:math:`\sqrt{100}` = 10 columns).
+
+  ``mtries`` is configured independently of ``col_sample_rate_per_tree``, but it can be limited by it. For example, if ``col_sample_rate_per_tree=0.01``, then there’s only one column left for each split, regardless of how large the value for ``mtries`` is.
+
+-  **Why does performance appear slower in Uplift DRF than in GBM?**
+
+  With DRF as well as Uplift DRF, depth and size of trees can result in speed tradeoffs.
+
+  By default, Uplift DRF will go to depth 20, which can lead to up to 1+2+4+8+…+2^19 ~ 1M nodes to be split, and for every one of them, mtries=sqrt(4600)=67 columns need to be considered for splitting. This results in a total work of finding up to 1M*67 ~ 67M split points per tree. Usually, many of the leaves don’t go to depth 20, so the actual number is less. (You can inspect the model to see that value.)
+
+  By default, GBM will go to depth 5, so there's only 1+2+4+8+16 = 31 nodes to be split, and for every one of them, all 4600 columns need to be considered. This results in a total work of finding up to 31*4600 ~ 143k split points (often all are needed) per tree.
+
+  This is why the shallow depth of GBM is one of the reasons it’s great for wide (for tree purposes) datasets. To make Uplift DRF faster, consider decreasing ``max_depth`` and/or ``mtries`` and/or ``ntrees``.
+
+  For both algorithms, finding one split requires a pass over one column and all rows. Assume a dataset with 250k rows and 500 columns. GBM can take minutes, while Uplift DRF may take hours. This is because:
+
+  -  Assuming the above, GBM needs to pass over up to 31\*500\*250k = 4 billion numbers per tree, and assuming 50 trees, that’s up to (typically equal to) 200 billion numbers in 11 minutes, or 300M per second, which is pretty fast;
+
+  -  Uplift DRF needs to pass over up to 1M\*22\*250k = 5500 billion numbers per tree, and assuming 50 trees, that’s up to 275 trillion numbers, which can take a few hours.
+
+
+Uplift DRF demo
```

Comment:
I know that IF, EIF, DeepLearning, GBM has this section right under the introduction. I would put this under Introduction also. Wdyt @hannah-tillman?

Reply:
Oh that makes more sense--keeps it consistent